addons/point_of_sale/models/product.py

```diff
@@ -36,6 +39,61 @@ def _unlink_except_active_pos_session(self):
             if self.with_context(product_ctx).search_count([('id', 'in', self.ids), ('product_tmpl_id.available_in_pos', '=', True)]):
                 raise UserError(_('You cannot delete a product saleable in point of sale while a session is still opened.'))
 
+    def get_product_info_pos(self, price, quantity, pos_config_id):
+        self.check_access_rights('read')
```

Comment:
not needed, if it traceback it is ok, as users in group pos_users, should have access to products